File: apps/frontend/src/modules/produtos/types/produto.ts
export interface Produto {
  id: number;
  numero?: number | null;
  descricao?: string | null;
  marca?: string | null;
  grupo?: number | null;
  nomegrupo?: string | null;
  patrimonio?: string | null;
  numeroserie?: string | null;
  acessorio?: string | null;
  mostracontrato?: string | null;
  status?: string | null;
  valorcompra?: number | null;
  valorestimado?: number | null;
  valorlimpeza?: number | null;
  quantidadereal?: number | null;
  unidade?: string | null;
  quantidadeestoque?: number | null;
  valorminimo?: number | null;
  valormensal?: number | null;
  valordiario?: number | null;
  tipo?: string | null;
  tabeladescontomensal?: number | null;
  nometabeladescontomensal?: string | null;
  descricaodetalhada?: string | null;
  locacao?: number | null;
  nomelocacao?: string | null;
}

export type ProdutoInput = Omit<Produto, 'id'> & { id?: number };

File: apps/backend/Entities/Produto.cs
namespace Gestor.Api.Entities;

public class Produto
{
    public int Id { get; set; }
    public int? Numero { get; set; }
    public string? Descricao { get; set; }
    public string? Marca { get; set; }
    public int? Grupo { get; set; }
    public string? Nomegrupo { get; set; }
    public string? Patrimonio { get; set; }
    public string? Numeroserie { get; set; }
    public string? Acessorio { get; set; }
    public string? Mostracontrato { get; set; }
    public string? Status { get; set; }
    public decimal? Valorcompra { get; set; }
    public decimal? Valorestimado { get; set; }
    public decimal? Valorlimpeza { get; set; }
    public decimal? Quantidadereal { get; set; }
    public string? Unidade { get; set; }
    public decimal? Quantidadeestoque { get; set; }
    public decimal? Valorminimo { get; set; }
    public decimal? Valormensal { get; set; }
    public decimal? Valordiario { get; set; }
    public string? Tipo { get; set; }
    public int? Tabeladescontomensal { get; set; }
    public string? Nometabeladescontomensal { get; set; }
    public string? Descricaodetalhada { get; set; }
    public int? Locacao { get; set; }
    public string? Nomelocacao { get; set; }
}
